src/bpftrace.cpp

```diff
@@ -316,6 +324,12 @@ int BPFtrace::add_probe(ast::Probe &p)
         // resolved function name are used in the probe.
         target = erase_prefix(func_id);
       }
+      else if ((probetype(attach_point->provider) == ProbeType::kprobe ||
+                probetype(attach_point->provider) == ProbeType::kretprobe) &&
+               !attach_point->target.empty())
```

Comment:
nit: merge into the previous case?

Reply:
I only unmerged the conditions for aesthetic reasons; since `kprobe` and `kretprobe` require the additional `attach_point->target` check, the indentation looks off to me. Will change though.